core/commands/refs.go

```diff
@@ -86,6 +89,12 @@ Note: List all referencess recursively by using the flag '-r'.
 			return
 		}
 
+		cached, _, err := req.Option("cached").Bool()
+		if err != nil {
+			res.SetError(err, cmds.ErrNormal)
+			return
+		}
+
```

Comment:
or at least, you might need less of it. In this case since you want a full listing of all local refs, you would just use the `Cached` flag to determine whether or not to error out on a `dag.ErrNotFound` type error from the dagservice `Get` method.